tests/test_util.py

```diff
@@ -6,17 +6,89 @@
 from canvasapi import Canvas
 from canvasapi.course import CourseNickname
 from canvasapi.user import User
-from canvasapi.util import combine_kwargs, obj_or_id
+from canvasapi.util import combine_kwargs, is_multivalued, obj_or_id
+from itertools import chain
+from six import integer_types, iterkeys, itervalues, iteritems
 from tests import settings
 from tests.util import register_uris
 
+try:
+    # in Python 2.x, iterator-returning zip function is "itertools.izip"
+    from itertools import izip as zip
+except ImportError:
+    # in Python 3.x, iterator-returning zip function is "zip" built-in
+    pass
+
 
 @requests_mock.Mocker()
 class TestUtil(unittest.TestCase):
 
     def setUp(self):
         self.canvas = Canvas(settings.BASE_URL, settings.API_KEY)
 
+    # is_multivalued()
+    def test_is_multivalued_bool(self, m):
+        self.assertFalse(is_multivalued(False))
+
+    def test_is_multivalued_integer_types(self, m):
+        for type in integer_types:
+            self.assertFalse(is_multivalued(type(1)))
+
+    def test_is_multivalued_str(self, m):
+        self.assertFalse(is_multivalued('string'))
```

Comment:
Since we import `unicode_literals`, this will always end up being a unicode string in both py2 and py3.

Not sure if we should remove it or leave it in. Certainly doesn't hurt to leave it.